README.md

```diff
@@ -2,33 +2,31 @@
 
 Each service might have a number of libraries available from each of the following categories:
 
-- [Client - New Releases](#client-new-releases)
-- [Client - Previous Versions](#client-previous-versions)
+- [Client](#client)
 - [Management](#management)
 
-### Client: New Releases
-
-New wave of packages that we are announcing as **GA** and several that are currently releasing in **preview** on `npm`. These libraries allow you to use and consume existing resources and interact with them. These libraries share a number of core functionalities found in the Azure Core package such as retries, logging, transport protocols, authentication protocols, etc. Learn more about these libraries by reading [the guidelines](https://azure.github.io/azure-sdk/typescript/guidelines/) that they follow.
-
-You can find the [most up to date list of all of the new packages on our page](https://azure.github.io/azure-sdk/releases/latest/index.html#javascript-packages)
+### Client
 
-> NOTE: If you need to ensure your code is ready for production use one of the stable, non-preview libraries.
+Given an Azure resource already exists, you would use the client libraries to consume it and interact with it.
+Most of these libraries follow the [Azure SDK Design Guidelines for JavaScript & TypeScript](https://azure.github.io/azure-sdk/typescript_introduction.html) and share a number of core functionalities such as retries, logging, transport protocols, authentication protocols, etc. Others will be updated in the near future to follow the guidelines as well.
 
-### Client: Previous Versions
-
-Last stable versions of packages that have been provided for usage with Azure and are production-ready. These libraries provide you with similar functionalities to the Preview ones as they allow you to use and consume existing resources and interact with them, for example: upload a blob. They might not implement the [guidelines](https://azure.github.io/azure-sdk/typescript_introduction.html) or have the same feature set as the Novemeber releases. They do however offer wider coverage of services.
+> NOTE: Some of these packages have beta versions. If you need to ensure your code is ready for production, use one of the stable, non-beta packages.
 
 ### Management
 
-Libraries which enable you to provision specific resources. They are responsible for directly mirroring and consuming Azure service's REST endpoints. You can recognize these libraries by `mgmt` or `arm` in their package names.
+Libraries which enable you to provision specific resources. You can recognize these libraries by `@azure/arm-` in their package names. These are purely auto-generated based on the swagger files that represent the APIs for resource management. These swagger files can be found in the [azure-rest-api-specs repository](https://github.com/Azure/azure-rest-api-specs/).
```

Comment:
Do we need to talk about how these packages are generated here in such detail? I think it would be best if this is mentioned in the contributing guide instead. For now, I think saying the following can be enough:
```suggestion
Auto-generated libraries which enable you to provision specific resources. You can recognize these libraries by `@azure/arm-` in their package names.
```

Reply:
Fixed in https://github.com/Azure/azure-sdk-for-js/pull/13255/commits/ac2995c55bc5464edcb481e2b6d8e714db9b29f2